core/SupportedBrowser.php

```diff
@@ -50,7 +50,7 @@ public static function checkIfBrowserSupported()
         $deviceDetector->parse();
         $client = $deviceDetector->getClient();
 
-        if ($client['type'] === 'browser' && self::browserNotSupported($client['short_name'], (int)$client['version'])) {
+        if (!empty($client) && $client['type'] === 'browser' && self::browserNotSupported($client['short_name'], (int)$client['version'])) {
```

Comment:
If device detector isn't able to detect any client the result of `getClient()` actually can be `null`, which then resulted in a notice here.